comms/hidapi/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# a6a622f (2016-01-28) is latest commit on master branch as of 2019-04-25;
+# contains macOS-related improvements since 0.8.0-rc1
+github.setup        signal11 hidapi a6a622ffb680c55da0de787ff93b80280498330f
+version             0.8.0
```

Comment:
I think that the version should be simplified to just `0.8.0`: even though it is prerelease, the upstream project has been updated too infrequently to justify using a more complicated, datestamped version. If and when an actual `0.8.0` release arrives, just increment the revision.

Reply:
Seems fine to me. I do similar things in other ports.